upload/library/Sidane/Threadmarks/Model/Threadmarks.php

```diff
@@ -3,6 +3,102 @@
 class Sidane_Threadmarks_Model_Threadmarks extends XenForo_Model
 {
 
+  public function getMenuLimit(array $thread, array $nodePermissions = null, array $viewingUser = null)
+  {
+    $this->standardizeViewingUserReferenceForNode($thread['node_id'], $viewingUser, $nodePermissions);
+    
+    $menulimit = XenForo_Permission::hasContentPermission($nodePermissions, 'permission_forum_sidane_tm_menu_limit'); 
```

Comment:
Makes sense but testing this from scratch again, with the menu limit set to 5 for the `Administrative` user group (I'm logged in as an admin), I see no threadmarks in the drop down menu.

Setting it to 'Unlimited` also shows no threadmarks.

I have the menu limit settings as the default everywhere else.

`$menulimit = XenForo_Permission::hasContentPermission($nodePermissions, 'permission_forum_sidane_tm_menu_limit');` is returning a boolean (`false` for my test user) but the subsequent code is expecting an integer.